### DISCUSSION TITLE
"HTTP Error 500.30 - ASP.NET Core app failed to start"

### DISCUSSION THREAD

**Discussion by ShifanilaSenthil** [abpframework/abp#18947] [Q&A]
I have added the source code of of Pro Modules using ABP Suite into my application , but when i set startup project as MyProjectName.web file , it shows "HTTP Error 500.30 - ASP.NET Core app failed to start" (app doesn't start).
How could we resolve this? Can u please instruct me with the steps to be done ?
![image](https://github.com/abpframework/abp/assets/106966267/af9c6cb1-17ad-40ee-9424-8410ae6e4f82)

**Comment by maliming** [comment]
hi

Please create a support ticket on https://support.abp.io/

Thanks.
